apps/backend/src/app/modules/maps/maps.service.spec.ts

```diff
@@ -103,7 +103,7 @@ describe('MapsService', () => {
       pass
         ? expect(await service.getMapAndCheckReadAccess({ mapID: map.id, userID })).toMatchObject(map)
         : await expect(service.getMapAndCheckReadAccess({ mapID: map.id, userID })).rejects.toThrow(ForbiddenException);
-    
+
```

Comment:
Random auto-formatting, don't want it in commits, I suggest just using Prettier and disabling any other formatters. I'm not sure why Prettier doesn't handle this, but if it doesn't, then so be it - would rather let Prettier handle this exclusively and avoid other autoformatter changes than have random whitespace changes like this.

Reply:
I think this one is removing the extra spaces.

Reply:
Oh you're right, it's the same issue with // prettier-ignore being ignored